packages/sdk-core/README.md

```diff
@@ -166,17 +174,18 @@ const { SinchClient } = require('@sinch/sdk-core');
 For various reasons (development phase, testing, network restrictions, ...), one may need to update the default API endpoint, pointing to production.
 Each API exposes dedicated parameters to override the default hostname and region:
 
-| API Name       | Parameter                          | Region             |
-|----------------|------------------------------------|--------------------|
-| Authentication | authHostname                       | N/A                |
-| SMS            | smsHostname                        | smsRegion          |
-| Conversation   | conversationHostname               | conversationRegion |
-|                | conversationTemplatesHostname      |                    |
-| Fax            | faxHostname                        | faxRegion          |
-| Voice          | voiceHostname                      | voiceRegion        |
-|                | voiceApplicationManagementHostname |                    |
-| Numbers        | numbersHostname                    | N/A                |
-| Verification   | verificationHostname               | N/A                |
+| API Name              | Parameter                          | Region             |
+|-----------------------|------------------------------------|--------------------|
+| Authentication        | authHostname                       | N/A                |
+| SMS                   | smsHostname                        | smsRegion          |
+| Conversation          | conversationHostname               | conversationRegion |
+|                       | conversationTemplatesHostname      |                    |
```

Comment:
Adding `N/A` for both empty cells (conversationTemplatesHostname, voiceApplicationManagementHostname) could have sense)

Reply:
Yes for `voiceApplicationManagementHostname`
No for `conversationTemplatesHostname`: I'll update it to `conversationRegion` as it is regionalized too and the same region must be used with the conversation API